visualization/app/codeCharta/codeCharta.component.ts

```diff
@@ -52,10 +52,16 @@ export class CodeChartaController {
 		}
 	}
 
-	tryLoadingSampleFiles(error: Error) {
+	tryLoadingSampleFiles(error: Error & { statusText?: string, status?: number }) {
```

Comment:
I don't understand how we have `statusText` and `status` available on the error object. The error object is thrown by `tryLoadingFiles` isnt it?

Reply:
It's from our http request.

Reply:
In `getFileDataFromQueryParam()`